tool.ts

```diff
@@ -260,14 +286,40 @@ export async function downloadTool(
             file.on('error', (err) => {
                 file.end();
                 reject(err);
-            })
+            });
         }
         catch (error) {
             reject(error);
         }
     });
 }
 
+//---------------------
+// Size functions
+//---------------------
+
+/**
+ * Gets size of downloaded file from "Content-Length" header
```

Comment:
Could it make sense probably to add some notes  for return value - that it could be NaN?